css-brochure-form/styles.css

```diff
@@ -0,0 +1,77 @@
+
+h1 {
+  font-family: 'Brandon Grotesque', sans-serif;
+  text-align: center;
+  color: #ec5c54;
+  font-size: 16px;
+}
+
+.container {
+  display: flex;
+  justify-items: center;
```

Comment:
Try removing this line and tell me what happens. You can send it to me in Slack if it isn't looking correct, that way you can update the pull request only when everything looks correct.